flowetl/tests/integration/conftest.py

```diff
@@ -0,0 +1,168 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# -*- coding: utf-8 -*-
+"""
+Conftest for flowetl integration tests
+"""
+import os
+import shutil
+
+from time import sleep
+from subprocess import DEVNULL, Popen
+from pendulum import now, Interval
+from airflow.models import DagRun
+
+import pytest
+import docker
+
+
+@pytest.fixture(scope="session")
+def docker_client():
+    """
+    docker client object
+    """
+    return docker.from_env()
+
+
+@pytest.fixture(scope="session")
+def flowetl_tag():
+    """
+    Get flowetl tag to use
+    """
+    return os.environ.get("FLOWETL_TAG", "latest")
+
+
+# pylint: disable=redefined-outer-name
+@pytest.fixture(scope="module")
+def flowetl_container(docker_client, flowetl_tag):
+    """
+    Fixture that starts a running flowetl container and
+    yeilds the container object.
+    """
+    container = docker_client.containers.run(
+        f"flowminder/flowetl:{flowetl_tag}", detach=True
+    )
+    # breif sleep to wait for backing DB to be ready
+    sleep(2)
+    yield container
+    container.kill()
+    container.remove()
+
+
+@pytest.fixture(scope="module")
+def airflow_local_setup_mdl_scope():
+    """
+    Init the airflow sqlitedb and start the scheduler with minimal env.
+    Clean up aftwards by removing the AIRFLOW_HOME and stopping the
+    scheduler. It is neccesery to set the AIRFLOW_HOME env variable on
```

Comment:
```suggestion
    scheduler. It is necessary to set the AIRFLOW_HOME env variable on
```

Reply:
Not sure I fully understand what this comment is saying. Does this mean bad things will happen if the `AIRFLOW_HOME` env var isn't defined? And where is the "somewhere else" that the comment refers to?

Reply:
`pytest` imports stuff from `airflow`. If `AIRFLOW_HOME` is not set *before* this happens then airflow creates it's home dir in `~` - if I then set the env var to the place I want it to be before running the `intidb` and `scheduler` we get two airflow homes and airflow gets confused...

There is an entry in their changelog about it but have not fully understood it - think it means you *have* to set it to have a non standard airflow home?

https://github.com/apache/airflow/blob/master/UPDATING.md#removal-of-airflow_home-config-setting